django_netjsonconfig/api/generics.py

```diff
@@ -10,7 +10,7 @@ class BaseSearchTemplate(ListAPIView):
     def get_queryset(self):
         name = self.request.GET.get('name', None)
         des = self.request.GET.get('des', None)
-        temp = self.template_model.filter(sharing='public')
+        temp = self.template_model.objects.filter(sharing='public')
```

Comment:
I think this commit should not be here, please remove that change from the PR, I suggest to squash the commits and remove this change (it may conflict with the other PR as well).